<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RAM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbVideoType">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>cam</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>video</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFps">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>560</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menu_help_info"/>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="menu_setting_exit"/>
   </widget>
   <addaction name="menu_setting"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_help_info">
   <property name="text">
    <string>Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="menu_setting_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
